### ISSUE TITLE
Authorization Header Lost

### ISSUE THREAD

**Issue by cawoodm** [a85/POSTMan-Chrome-Extension#3547]
## App Details:

```
Postman for Windows
Version 5.2.0
win32 10.0.14393 / x64
```

## Issue Report:
Recently moved to the windows (not chrome) app. Whenever I change the credentials on the Headers tab to a variable value:
![auth1](https://user-images.githubusercontent.com/498834/30731706-408e9328-9f6f-11e7-84e7-4de07692b2f8.png)

they get changed back to whatever is on the Authorization tab when the request is sent
![auth2](https://user-images.githubusercontent.com/498834/30731707-40a4a528-9f6f-11e7-8ffd-82fb2d59ccc8.png)

It only happens when the request is sent or saved.

When we set the Authorization to "None" it does not happen.

Basically whatever is on the authorization tab is overwriting the headers even if the headers field is changed after the authorization tab. This was no so in the Chrome version.

**Comment by kamalaknn** [comment]
@cawoodm this is because the `Authorization` will always override your header. If you want to use a different `Authorization` header, you would have to do by setting `Authorization` to `No Auth`, which you have noticed.

However, Postman should not automatically replace or add a `Header` to the request itself. We will be fixing it in the next version.

I'll update this thread when this fix is rolled out.

**Comment by kamalaknn** [comment]
Hi @cawoodm 

Starting from `5.3.0` Postman will not add or replace the headers saved with your request. Postman will also warn if you have a header that might be overridden by Postman like this

![screen shot 2017-10-09 at 1 25 45 pm](https://user-images.githubusercontent.com/8401064/31329514-21bcd2ac-acf8-11e7-83af-8c8c1b07c521.png)

This should make sure the header you added was not lost while sending the request.

Hope this helps. I am closing this issue, feel free to update this thread if you are not seeing this behavior.

**Comment by Ajay-Girolkar** [comment]
> @cawoodm this is because the `Authorization` will always override your header. If you want to use a different `Authorization` header, you would have to do by setting `Authorization` to `No Auth`, which you have noticed.
> 
> However, Postman should not automatically replace or add a `Header` to the request itself. We will be fixing it in the next version.
> 
> I'll update this thread when this fix is rolled out.

Thanks. Solve my issue
